### PR TITLE
Skeleton pages for Home and Filter

### PR DESCRIPTION
**Pull request by payalahuja98**

### CODE REVIEW

**Review by annazelenka — APPROVED** [review]
Looks good!

**Review by khalludi — COMMENTED** [review]
A few things. First of all, where is your Dart code that does the navigation? I don't see that anywhere.

The .metadata file has changed saying that it switched from the beta channel of Flutter to the dev channel. You should switch your local flutter version to the beta channel. Look up how to do this.

A few files are being replaced/added which shouldn't be such as the .metadata file, the .packages, and the pubspec.yaml. This tells me that you tried to do a `flutter create ice_cream_social_app` inside the repository after cloning or maybe you copy-pasted these files from another project. Either way, these files shouldn't be completely replaced. The .iml file should also be git ignored.

In summary, please make a new branch from master, make sure your flutter version is following beta, do a pub get, and make sure the application runs. THEN add your navigation Dart files under /lib and add the custom font to the bottom of pubspec.yaml without completely copy-pasting from your side project. 

I am closing this PR. Please create a new branch and submit a new PR.
